jenkins/test/Jenkinsfile

```diff
@@ -13,23 +13,27 @@ pipeline {
     stages {
         stage("Build") {
```

Comment:
My comments related to rationale behind having a separate `Build` stage while performing build operation also as a part of subsequent stage are applicable also here.

Reply:
I've removed separate build stage, so now this pipeline contains only test stage which executes `mvn clean test`.